conda_forge_tick/migrators.xsh

```diff
@@ -144,10 +142,17 @@ class Version(Migrator):
                         continue
             else:
                 hash = hash_url(url, hash_type)
-            p = '  {}:\s*[0-9A-Fa-f]+'.format(hash_type)
-            n = '  {}: {}'.format(hash_type, hash)
-            if platform is not None:
-                p += '\s*(#.*)\[{}\](?(1)[^\(\)]*)$'.format(platform)
+            m = re.search('\s*{}:(.+)'.format(hash_type), line)
+            if m is not None:
```

Comment:
It is generally eaiser to follow positive logical expressions.  I would recommend swapping this to be `if m is None`